### PR TITLE
make AuthorsMixin fields optional

### PR DESCRIPTION
**Pull request by petergx**

### CODE REVIEW

File: armstrong/core/arm_content/tests/mixins/authors.py [hunk 1]
```diff
@@ -5,6 +5,8 @@ from .._utils import *
 from ..arm_content_support.models import SimpleMixedinAuthorModel
 from ..arm_content_support.models import SimpleProfile
 
+from ..arm_content_support.forms import SimpleMixedinAuthorForm
```
Comment (tswicegood) [inline armstrong/core/arm_content/tests/mixins/authors.py:8]: Where is `SimpleMixedinAuthorForm`?

**Comment by tswicegood** [comment]
Just verified that this does not run.  Needs to have the `forms` module added before it can be merged.

File: armstrong/core/arm_content/tests/mixins/authors.py (hunk 1 above)
Comment (petergx) [inline armstrong/core/arm_content/tests/mixins/authors.py:8]: oops forgot to git add it. it's there now.

**Comment by tswicegood** [comment]
Looking at this again...

**Pull request merged**